src/components/Public/MenuLevel.vue

```diff
@@ -38,6 +38,7 @@ export default {
 </script>
 <style lang="scss" scoped>
 @import "@/assets/sass/_0.declare.scss";
+/* stylelint-disable no-descending-specificity */
```

Comment:
Hi, can we support the current style lint rules? This is an easy rule to support.